java/code/src/com/redhat/rhn/frontend/xmlrpc/chain/test/ActionChainHandlerTest.java

```diff
@@ -183,7 +180,7 @@ public void testAcCreateActionChain() {
     public void testAcCreateActionChainFailureOnEmptyName() {
         try {
             this.ach.createChain(this.admin, "");
-            fail("Expected exception: " +
+            fail(EXPECTED_EXCEPTION +
                  InvalidParameterException.class.getCanonicalName());
         }
         catch (InvalidParameterException ex) {
```

Comment:
This is an old way of writing this type of checks.  With modern junit, here and in the other similar places, this could just be:

```java
        assertThrows(InvalidParameterException.class, () -> this.ach.createChain(this.admin, ""));
```

and junit will take care of writing a meaningful message.